api/types/mount/mount.go

```diff
@@ -8,17 +8,21 @@ const (
 	TypeBind Type = "bind"
 	// TypeVolume VOLUME
 	TypeVolume Type = "volume"
+	// TypeTmpfs TMPFS
```

Comment:
:) We should just group these into a single comment above the `const` block describing what they are for rather than having these redundant doc strings.